### PR TITLE
feat: Wishlist

### PR DESCRIPTION
**Pull request by hiteshverma27**
Features added 
- Add to wishlist 
- Remove from wishlist 
- Move to cart 
- Move to wishlist
mobile (4) (1)

### CODE REVIEW

**Review by ishitakeshawani — APPROVED** [review]
Clean and readable code
On the shop now button, at the homepage when the user does hover the text is not visible. Please fix it.

File: src/components/Navigation/Navigation.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ function Navigation() {
       </Link>
       <ul className="flex-space_between-center" id="nav-icons">
         <li className="mx-2">
-          <Link to={"/"} href="/Pages/login.html">
+          <Link to={"/"}>
             <button className="btn-primary-confirm">Login</button>
           </Link>
```
Comment (rohanmathur91) [inline src/components/Navigation/Navigation.js:19]: dont use button inside link, either use link or button + useNavigate to navigate, depending upon the use case

File: src/components/Navigation/Navigation.js [hunk 2]
```diff
@@ -31,17 +31,21 @@ function Navigation() {
...
                 favorite_border
               </span>
-              <span className="badge badge-red"></span>
+              {wishlist.length ? (
+                <span className="badge badge-red">
+                  {wishlist.length ? wishlist.length : null}
+                </span>
+              ) : null}
             </button>
           </Link>
```
Comment (rohanmathur91) [inline src/components/Navigation/Navigation.js:45]: same as above

**Review by rohanmathur91 — APPROVED** [review]
- code is clean and readable
- Good that you have done side-effects inside useEffect

**Pull request merged**